ui/app/styles/components/chart-container.scss

```diff
@@ -20,7 +20,7 @@
 .single-chart-grid {
   display: grid;
   grid-template-columns: 1fr 0.3fr 3.7fr;
-  grid-template-rows: 0.5fr 1fr 1fr 1fr 0.25fr;
+  grid-template-rows: 0.5fr 1fr 1fr 0.5fr 0.25fr;
```

Comment:
I might be mis-remembering but I think a row here could be deleted all together since the timestamp lived in it's own row. But I trust your judgement if you think this looks fine in the app!